File: cs/Iris/Data/IrisImage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Media.Imaging;
using System.IO;

namespace Iris.Data
{
    public class IrisImage
    {
        public static readonly string baseFolder = Directory.GetCurrentDirectory() + "\\images\\";

        public string name;
        public BitmapImage bmp;

        public IrisImage()
        {

        }

        public IrisImage(string fileName)
        {
            bmp = new BitmapImage(new Uri(baseFolder + fileName, UriKind.Absolute));
        }

        public IrisImage(string imageName, string filePath) : this(filePath)
        {
            this.name = imageName;
        }
    }
}

File: java/Iris/src/com/rokuan/iris/data/IrisImage.java
package com.rokuan.iris.data;

public class IrisImage extends IrisFile
{
	private static final String baseFolder = "images/";

	public String name;

	public IrisImage()
	{

	}

	public IrisImage(String filePath)
	{
		this.setFilepath(filePath);
	}

	public IrisImage(String imageName, String filePath)
	{
		this(filePath);
		this.name = imageName;
	}
	
	@Override
	public void setFilepath(String path){
		super.setFilepath(baseFolder + path);
	}
}
